### PR TITLE
[T6A3][F11-C1] Louie Tan Yi Jie

### PR DESCRIPTION
**Pull request by louietyj**
Ready for review.

### CODE REVIEW

**Comment by louietyj** [comment]
Please ignore broken CI builds if you see them - rebased the CI files away so as not to pollute the PR.

File: src/seedu/addressbook/storage/Storage.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    public abstract void save(AddressBook addressBook) throws StorageOperationException;
+    
+    /**
+     * Loads data from this storage file.
+     *
+     * @throws StorageOperationException if there were errors reading and/or converting data from file.
+     */
+    public abstract AddressBook load() throws StorageOperationException;
+    
+    public abstract String getPath();
```
Comment (pixelducky) [inline src/seedu/addressbook/storage/Storage.java]: `Storage` class can be made more cohesive by moving `getPath()` method back to `StorageFile` class

**Comment by pixelducky** [comment]
@louietyj  Some comments added. Please close the PR after reading comments 👍

File: src/seedu/addressbook/storage/Storage.java (hunk 1 above)
Comment (pixelducky) [inline src/seedu/addressbook/storage/Storage.java]: Having `getPath` here is totally valid in this case. My bad. Please ignore my previous comments. 
p/s: this is what happens when there's too much going on during midterms week :-X

**Pull request closed without merging**